### ISSUE TITLE
Vaultless Vaults

### ISSUE THREAD

**Issue by supercolinoid** [Iskallia/Vault-public#825]
**Describe the bug**

My game recently crashed when i was trying to access the storage modules (tier 2) from rftoolsstorage directly, saying something like "could not access storage at 000000000000". I reopened my world and all my rftoolsstorage and my cryo chamber was empty. I turned on cheats to get most of my stuff back, and started making backups whenever I loaded my world. Now whenever I enter the vault using my vault portal, I spawn way up in the air (probably at around y200). I fall all the way down to 64 and I'm left in a flat world of vault bedrock. The exact coordinates I spawn at are at about x4000 z4000. My guess as to what is happening is that when I reloaded my world after the crash, it reset whatever counter it used for where i spawn, so the vault is generating somewhere far away, but I'm being spawned back near the start. Luckily I made a backup before entering so I havent lost my stuff in the vault.

**To Reproduce**
1. Access rftoolsstorage storage module
2. Have it crash?
3. Load world or backup
4. Enter vault

**Expected behavior**
Vaults were expected to be vaultful.

**Screenshots or Crash Report**
If applicable, add screenshots or crash reports to help explain your problem.
![Screenshot (5)](https://user-images.githubusercontent.com/74925713/154863469-9065fa71-c6b2-41a6-b401-0d001cb06d3e.png)
Where I usually spawn when i first enter. When i enter the vault again im spawned 4000 further to the right like normal from the last time.

**Info (please complete the following information):**
- World Type: Single Player
- The_Vault Version 1.12.4

**Additional context**
My pc does sometimes lag but doesn't usually crash.

**Comment by supercolinoid** [comment]
I did some more testing and found out that the vaults aren't generating at all. I made a world to test where vaults generated and got the average distance of x8,189 blocks between the center of each vault. In photo 1 I am standing in the center of the vault that's 2 vaults before the crash, then in the second photo i teleport 8189 blocks further and am almost in the exact center of the vault before the crash. In the third photo I teleported another 8189 blocks further and found nothing nearby (the hole was dug by me). 

I tried going further by about 16 more vaults and still found nothing. I tried checking the original spawn to see if it was spawning new vaults over the old ones, but they haven't changed. If anybody knows a way to fix this or reset the vault dimension it would be greatly appreciated.

(Second to last vault)
![Screenshot (10)](https://user-images.githubusercontent.com/74925713/155775258-8812122c-8b69-4d7d-b8e1-dfd138db4838.png)

(Last vault)
![Screenshot (8)](https://user-images.githubusercontent.com/74925713/155775302-95ddefe7-2920-433f-b268-2ca175b9dc33.png)

(Where my next vault would be)
![Screenshot (11)](https://user-images.githubusercontent.com/74925713/155775403-0638f478-b6a7-49c0-a9dc-d737ab5f1f2b.png)
